src/gwmock/cli/adapter_orchestration.py

```
@@ -686,6 +695,101 @@ def _event_starts_before_segment_end(self, parameters: Mapping[str, Any]) -> boo
+        # tests passed. Letting the `AttributeError` fall into the handler below produces the same
+        # conservative `None` *and* warns once, which is the codebase's stated rule: the fallback
+        # is the previous behaviour, and its cost is reported rather than silent.
+        try:
+            return self.signal_adapter.post_coalescence_duration(
+                parameters,
+                sampling_frequency=float(self.sampling_frequency.value),
+                minimum_frequency=self.minimum_frequency,
+                waveform_arguments=self.waveform_arguments,
+                waveform_options=self.waveform_options,
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)